modules/stardoc/0.8.0/presubmit.yml

```
@@ -0,0 +1,15 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
+  - macos
+  - windows
+  bazel:
+  - 8.x
```

```suggestion
  - 8.x
  - 7.x
```
Please test with all versions for which this module claims compatibility.